### DISCUSSION TITLE
How to configure the URL for the web server behind the Git icon?

### DISCUSSION THREAD

**Discussion by smuehlst** [JanDeDobbeleer/oh-my-posh#4910] [Troubleshoot] [answered]
Hi,

I'm just getting started with Oh My Posh, and the feature I'm most excited about is the possibility to click into the prompt to open the Windows Explorer or the Git web server.

Regarding Git this works fine for example with a public repository that is hosted on GitHub, but we have an internal GitLab server that is only configured for access via HTTP, but not HTTPS. Clicking the Git icon tries to open a `https://` URL and this fails therefore.

Is it possible to configure the way the URL for the Git web server is built? I checked the documentation, but I could not find something about how this URL is created.

**Comment by JanDeDobbeleer** [comment, marked as answer]
We [force](https://github.com/JanDeDobbeleer/oh-my-posh/blob/aee7ccab5dd5526797a97391b684608536bd0a35/src/segments/git.go#L473) to `https` indeed. There's no functionality to adjust that as I (incorrectly) assumed nobody would dare to use `http` in 2020 😓

**Reply by smuehlst** [reply]
Thanks for clarifying this. Of course I wouldn't want to use `http` with public Git web servers, but I think for small Intranets it is still useful.
